RubeesEat/Pages/SplitBill.cshtml

```diff
@@ -19,9 +22,10 @@ Split bill:
             <option value="@person.FirstName @person.LastName" data-id="@person.Id">@person.FirstName @person.LastName </option>
         }
     </select>
-    <input id="addPersonButton" type="button" onclick="AddPerson()" value="Add Person"><br/>
+    <input id="addPersonButton" t-action="addPersonButton" t-click-behavior="Click" type="button" onclick="AddPerson()" value="Add Person"></input><br/>
 
     <button id="splitBillButton" type="button" onclick="validate()">Split the bill</button><br>
     <text id="placeForErrorMessage" style="color: red;"></text>
 </form>
+<script type="text/javascript">const isProduction = @env.IsProduction().ToString().ToLower();</script>
```

Comment:
This will only apply the change for this one page, so if we want to have other pages with JS as well, this won't work. I would suggest moving this to the layout and inverting the variable. So we have a variable `isTest` which we set if the environment is `Test`, otherwise no JS is injected.